Engine/Code/Engine/Graphics/Mesh.cpp

```diff
@@ -3,6 +3,7 @@
 #include "Buffers/VertexArrayObject.h"
 #include "Buffers/IndexBuffer.h"
 #include "MeshBuilder.h"
+#include "Vertex.h"
```

Comment:
I don't see a use of vertex.h here anywhere not sure why you need the include

Reply:
VertexBufferLayout is delcared in the vertex file.